guacamole-ext/src/main/java/org/apache/guacamole/token/StandardTokens.java

```diff
@@ -21,9 +21,10 @@
 
 import java.text.SimpleDateFormat;
 import java.util.Date;
+import java.util.Map;
+import java.util.Set;
 import org.apache.guacamole.net.auth.AuthenticatedUser;
 import org.apache.guacamole.net.auth.Credentials;
-
```

Comment:
Add this line back in - style-wise, there should be a line between the imports and the class documentation.